### ISSUE TITLE
No error thrown when methods are called without loading library

### ISSUE THREAD

**Issue by ayZagen** [PeculiarVentures/pkcs11js#83]
Process exits directly when pkcs methods are called without loading library. Even though it is wrong usage process should not exit an exception could be thrown.

Reproducing code:
```js
const pkcs11js = require("pkcs11js");

try{
  const pkcs11 = new pkcs11js.PKCS11();

  const module_info = pkcs11.C_GetInfo(); //call method without pkcs11.load
  console.log(module_info)
}catch(err){
  console.log("should print")
}finally{
  console.log("should print")
}
```

**Comment by microshine** [comment]
I'll fix it

**Comment by microshine** [comment]
The new version is available `pkcs11js@1.2.5`
